data/hai/hai reveal 5 philosophers.txt

```diff
@@ -309,201 +313,78 @@ mission "Hai Reveal [B03] Contact Giti"
-	on enter "Deep Space 7C"
-		log `Passed through the wormhole in Syndicate space to a region just outside the Milky Way galaxy. It looks like a good place to hide.`
-		dialog `It takes your computer a few seconds to triangulate, but it says you've left the Milky Way galaxy and are now just beyond the edge of the known galactic disc. Your computer is identifying the system name as "Deep Space 7C" from old astronomical surveys done in the early 22nd century. This is definitely a good place to hide. It is past time to return to Giti now and discuss what to do next.`
 	on offer
-		event "reveal devil-run"
+		event "reveal devil-hide"
 		conversation
 			`Once safely parked and hooked into the relay network from Canyon you check timezones and place a call to Giti. Astonishingly, you manage to get a remarkably clean connection and don't have to leave a message. Perhaps by fluke it's night in all the most populous places between here and there. She's at her desk when she answers, and it occurs to you for the first time that a research console on a Navy grade connection probably has pretty good bandwidth. "Oh it's a relief to see you Captain. There's been so much going on, I feared you'd been caught up and detained with all that blockade stuff that all over the news right now. You're lucky though, it's quite late here and I was just about to pack up."`
 			choice
 				`	"I was a little caught up, actually."`
 				`	"What do you know about the blockades?"`
 			`	As you draw breath to speak she cuts you off. "I actually don't have a lot of time or I'll miss my ride, so we'll skip the pleasantries. Obviously if you're calling me back you want to know what I found about the notation. That too is something you have good timing on, I was delayed on that until just a couple days ago. Now where was that file... ah here it is.`
-			`	"So, the notation is actually quite dense, and is mostly some probably meaningless prose about something called 'Elenchus'. It sounds aspirational. However, it also refers to a secret gateway. I'm sending you some coordinates." You receive a simple text message a moment later with some system coordinates in it.`
+			`	"The reason I had trouble with this last time you were here is that some of these symbols are from Set 3-A, which were never translated. Fortunately, the researchers in Deneb found a Rosetta Stone of sorts, so I know what it says. The first part is some meaningless prose about something called 'Elenchus', which sounds aspirational. I have its location, in a system called Devil-Hide, far outside of our galaxy to the north-west. The second part contains a riddle, which gives the location of another system, Devil-Run, somewhere near the galactic core. I won't know the exact location until I solve the riddle: 'Where does the soul sleep?'"`
```

Comment:
I have also attached a suggestion to preserve the translation practicalities in accordance with the remainder of the necessary changes.
```suggestion
			`	"So, the notation is actually quite dense which makes perfect translation an uncertain prospect. The first part is some meaningless prose about something called 'Elenchus', which sounds aspirational. I have its location, in a system called Devil-Hide, far outside of our galaxy to the north-west. The second part contains a riddle, which gives the location of another system, Devil-Run, somewhere near the galactic core. I won't know the exact location until I solve the riddle: 'Where does the soul sleep?'"`
```

Reply:
The wormhole's type and location changed, so I had to change the justifications for people's actions.

Reply:
That explains the latter part and so on, which all made sense. I don't think that applies to changing why the translation was difficult though. That seems to be just adding complexity and ideas which are unnecessarily distracting.

Reply:
The, "Where does the soul sleep?" bit is her initial mistranslation of "0.184 stiradian hyperspace aperture window." She assumed it was a riddle because it sounded like one. The player is sent on a wild goose chase to find the meaning of the riddle, only to discover the correct aperture window size instead.